theme/less/_content.less

```diff
@@ -2,3 +2,31 @@ main article img
 {
   max-width: 100%;
 }
+
+article
+{
+  color: #444444;
+}
+
+article p, article ul, article ol
+{
+  font-size: 14px;
+  margin: 0 0 20px 0;
+  line-height: 1.5;
+}
+
+article h1
+{
+  font-size: 35px;
+  font-weight: normal;
```

Comment:
Suggestion: Let's use numerical representation(e.i. `400`) of the `font-weight`, it's more handy when using external fonts, so you know exacly the weight of the fonts you need to import.